Example/Artsy_Authentication/ARViewController.m

```diff
@@ -1,7 +1,13 @@
 #import "ARViewController.h"
+
+#import <Pods_Artsy_Authentication/Artsy_AuthenticationKeys.h>
+
 @import Artsy_Authentication;
 
 @interface ARViewController ()
+
+@property (nonatomic, strong) ArtsyAuthentication *auth;
```

Comment:
This needs to be retained, since it retains an `ACAccountStore`, which must live for the duration of all requests.